Winapp2.ini

```diff
@@ -8820,6 +8905,27 @@ DetectFile=%ProgramFiles%\HomeCinema
 Default=False
 FileKey1=%ProgramFiles%\HomeCinema\PowerProducer\Template\Cyberlink\frame|*tmp
 
+[Homeworld: Deserts of Kharak Backups *]
+Section=Games
+Detect=HKCU\Software\Valve\Steam\Apps\281610
+Default=False
+Warning=You will not be able to restore corrupted saves.
+FileKey1=%ProgramFiles%\Steam\steamapps\common\Deserts of Kharak\out\backup|*.*|RECURSE
+
+[Homeworld Remastered Collection *]
+Section=Games
+Detect=HKCU\Software\Valve\Steam\Apps\244160
+Default=False
+FileKey1=%ProgramFiles%\Steam\steamapps\common\Homeworld\Homeworld2Classic\Bin\LOGFILES|*.*
+FileKey2=%ProgramFiles%\Steam\steamapps\common\Homeworld\HomeworldRM\Bin\LOGFILES|*.*
+
+[Homeworld Remastered Collection Cache *]
+Section=Games
+Detect=HKCU\Software\Valve\Steam\Apps\244160
+Default=False
+FileKey1=%ProgramFiles%\Steam\steamapps\common\Homeworld\Homeworld2Classic\Bin\CACHE|*.*|RECURSE
+FileKey2=%ProgramFiles%\Steam\steamapps\common\Homeworld\HomeworldRM\Bin\CACHE|*.*|RECURSE

```

Comment:
Can FileKey1+2 be merged?

[Homeworld Remastered Collection \*]
```FileKey1=%ProgramFiles%\Steam\steamapps\common\Homeworld\Homeworld*\Bin\LOGFILES|*.*```
[Homeworld Remastered Collection Cache \*]
```FileKey1=%ProgramFiles%\Steam\steamapps\common\Homeworld\Homeworld*\Bin\CACHE|*.*|RECURSE```